### DISCUSSION TITLE
关于世界设定的一些疑问和讨论（正经）

### DISCUSSION THREAD

**Discussion by a76031227** [share121/inter-knot#4763] [绝区零]
首先说明，个人认同 一些因为很酷 所有不算特别合理也可以接受的设定的。

例如，为了体现复古潮流 ，在明明网络和各种移动及非移动终端都有很高算力的情况下，唱片行、录像店、游戏厅这些本在千禧年前后那个社会背景下才能流行起来的元素都依旧流行。

因为很酷，而且有情怀，所有合理不合理都无所谓。

![ccd80f87c9177f3e6d12612f36cf3bc79f3d56fb](https://github.com/user-attachments/assets/8687c471-49e7-4aed-8e60-4055aa9d978e)

出处P站：[Rimeye](https://www.pixiv.net/users/76763955)

但是，有一些设定，让我有点迷糊，是从逻辑上理解不了（可能是是还没展开暂时没说明）。

首先是关于AI的科技水平，以及在日常应用中的普及程度的。

从目前展现出来的剧情来看，这个世界是普遍存在类人形的机械人（？）例如[汀曼大师](https://wiki.biligame.com/zzz/%E6%B1%80%E6%9B%BC%E5%A4%A7%E5%B8%88)、[乔普师傅](https://wiki.biligame.com/zzz/%E4%B9%94%E6%99%AE%E5%B8%88%E5%82%85)、[乔普小子](https://wiki.biligame.com/zzz/%E4%B9%94%E6%99%AE%E5%B0%8F%E5%AD%90)、[埃珐](https://wiki.biligame.com/zzz/%E5%9F%83%E7%8F%90)、还有一个不记得名字的按摩师傅。因为没看过相关的具体介绍，不清楚这些角色是强AI的机械人还是经过义体改造的人类，所有暂不讨论。

但是还有更普遍的，明显是人造机械的--邦布。剧情里出现的很多的邦布的智能都很高，甚至有些已经明显有自我意识，会思考我、我该做什么、什么工作适合我等各种问题了。

![f31d2543d1b1d1257e6cb965e593d625](https://github.com/user-attachments/assets/bc0e923e-37ef-4ce1-86b2-af76aacf8384)

出处X：[@Bea_Lenoir](https://twitter.com/Bea_Lenoir)

而且还有更者的，是青衣这个角色，按目前的资料，个人的理解青衣无论是身体还是人格（灵魂）是人造产物， 当然也可能是什么伏笔吧。

通过以上的信息，大致可以判断这个世界是普遍存在强AI的存在的，而且大家都已经习以为常。

![sample_7b98445f3917ac57dcef639ca7d0fd83](https://github.com/user-attachments/assets/414e75fe-d101-4241-b68d-222c4a85780d)

出处X：[@arikuigames1105](https://x.com/arikuigames1105)

那么有个疑问是，从格莉丝的PV以及剧情里的体现，白祇重工的空洞用作业智能机械是很尖端的产物。也是作为天才的格莉丝和前社长研发出来的产品，是白祇重工的企业优势。

但是，从个人观感来说，白祇重工的这些尖端智能产品，好像还不如在新艾利都常见的邦布智能程度高啊。个别邦布的自我意识都都能通过图灵测试了啊。这个是剧情展现的问题 ，还是设定上的BUG呢？

![sample_e6c3f2f88e1a61c48ddbfe5c32883ad2](https://github.com/user-attachments/assets/3a3ae40c-2b29-4f27-bb04-f664cf153caa)

出处P站：[女児入道](https://www.pixiv.net/users/97203081)

然后还有一点我觉得可能不太符合逻辑的，就是空洞离普通居民的生活区太近了。从剧情上的描述来说，空洞都是不稳定的，比活火山还危险的东西，居然附近会建立起新艾利都这样的大型城市？你们新艾利都的人心都这么大的嘛.... 

不过这点可以用“因为这很酷”来解释吧~

**Comment by March-777th** [comment]
白祇重工的机械都能跟烂尾楼谈恋爱了<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >与其说智能低不如说心智不成熟

**Comment by CartersofBarbel** [comment]
这个世界就是一个模拟电路的if线，所以才有显像管电视📺这种老掉牙的模电产品，还有示波器录像带什么的。而高级人工智能包括青衣其实都是邦布的进阶版。现在的智能邦布都是玛瑟尔集团不断改善邦布性能的结果。你要问没有数字电路怎么会诞生fairy这种超级ai，那只能说这个世界不过都是一个黑客帝国罢了，四个ai被用来监视文明的演化（doge）

**Comment by Robin-txdy** [comment]
头好痒，感觉要长脑子了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >
